changelog/unreleased/pull-2750

```diff
@@ -0,0 +1,5 @@
+Enhancement: Add --min-packsize and --file-read-concurrency
+
+In order to tune restic for large installs we need to be able to configure --min-packsize and --file-read-concurrency.
```

Comment:
I would change this to "in order to tune restic for special sitations"

Reply:
Agreed, done.